ext/stringio/stringio.c

```diff
@@ -1366,11 +1368,13 @@ strio_getline(struct getline_arg *arg, struct StringIO *ptr)
 
 /*
  * call-seq:
- *   strio.gets(sep=$/, chomp: false)     -> string or nil
- *   strio.gets(limit, chomp: false)      -> string or nil
- *   strio.gets(sep, limit, chomp: false) -> string or nil
+ *   gets(sep = $/, **line_opts) -> string or nil
+ *   gets(limit, **line_opts) -> string or nil
+ *   gets(sep, limit, **line_opts) -> string or nil
```

Comment:
It looks like the only keyword argument that these method accept is `chomp:`, so I think it's clearer to explicitly state that rather than having a generic `**line_opts`.

Reply:
Fixed (9 places).